contracts/YVaultAssetProxy.sol

```diff
@@ -60,9 +92,7 @@ contract YVaultAssetProxy is WrappedPosition {
         (uint256 localUnderlying, uint256 localShares) = _getReserves();
         // Calculate the total reserve value
         uint256 totalValue = localUnderlying;
-        uint256 yearnTotalSupply = vault.totalSupply();
-        uint256 yearnTotalAssets = vault.totalAssets();
-        totalValue += ((yearnTotalAssets * localShares) / yearnTotalSupply);
+        totalValue += _yearnDepositConverter(localShares, true);
```

Comment:
`_yearnDepositConverter(localShares, true)` would be slightly smaller than `vault.withdraw(localShares)` due to rounding errors, thus `totalValue` would be smaller than the exact value.  This can lead to `mintAmount` being greater than it should be.  So, in theory, repeating reserveDeposit() and reserveWithdraw() could make a profit.  It is not yet clear how big the profit can be, and whether it can be larger than the gas cost or not.